tests/test_topog.py

```diff
@@ -1,45 +1,51 @@
 # unit test for Topog.py functionality
 
-from gridtools import TopogObj
+from FMSgridtools import TopogObj
+from FMSgridtools import get_provenance_attrs
 import numpy as np
 import pytest
 from pathlib import Path
 from os import remove
 
-def test_write_one_tile():
-    out_file = "test_topog_single_tile.nc"
+def test_generate_rectangular_basin_single_tile():
+    out_file = "test_topog_rectangular_basin_single_tile.nc"
     test_topog = TopogObj(output_name=out_file,
                           ntiles = 1,
-                          nx = 16,
-                          ny = 16,
-                          depth = np.zeros((16, 16)),
+                          global_attrs = get_provenance_attrs(),
+                          nx = {'tile1': 16},
+                          ny = {'tile1': 16},
                           x_refine = 1,
                           y_refine = 1,
                           scale_factor = 1)
+    test_topog.make_rectangular_basin(bottom_depth=128)
     test_topog.write_topog_file()
-    assert Path(out_file).exists()
-    remove(out_file)
 
-def test_write_multi_tile():
-    out_file = "test_topog_multi_tile.nc"
+def test_generate_rectangular_basin_multi_tile():
+    out_file = "test_topog_rectangular_basin_multi_tile.nc"
     test_topog = TopogObj(output_name=out_file,
                           ntiles = 6,
-                          nx = 16,
-                          ny = 16,
-                          depth = np.zeros((16, 16, 6)),
+                          global_attrs = get_provenance_attrs(),
+                          nx = {'tile1': 16,
+                                'tile2': 16,
+                                'tile3': 16,
+                                'tile4': 16,
+                                'tile5': 16,
+                                'tile6': 16},
+                          ny = {'tile1': 16,
+                                'tile2': 16,
+                                'tile3': 16,
+                                'tile4': 16,
+                                'tile5': 16,
+                                'tile6': 16},
                           x_refine = 1,
                           y_refine = 1,
                           scale_factor = 1)
+    test_topog.make_rectangular_basin(bottom_depth=128)
     test_topog.write_topog_file()
-    assert Path(out_file).exists()
```

Comment:
why was this removed?

Reply:
I don't know honestly, now its added back in along with some output checking for the written files.